src/typings/dts-jsdoc.d.ts

```diff
@@ -11,4 +11,10 @@ declare interface ITemplateConfig {
      */
     debug?: boolean;
     access?: ("public" | "private" | "protected" | "package");
+    /**
+     * See [tsd-jsdoc#100](https://github.com/englercj/tsd-jsdoc/issues/100).
+     * The 'exported' generation strategy is activated through an option.
+     * In order not to break backward compatibility, the 'documented' generation strategy remains the default.
```

Comment:
That's a question: should the 'documented' strategy remain the default?
@englercj, what's your opinion?

Reply:
I think I would prefer using the exported strategy by default and we will do a major version bump.

Reply:
Nice!
I leave it optional and experimental for the moment in that PR.
Up to you to make it the default in a new version when you feel it is stable enough.